src/data/[provider]-spark-retrieval-timings-summary.json.js

```diff
@@ -0,0 +1,17 @@
+import pRetry from 'p-retry';
+import { jsonFetcher } from "./json-fetcher.js";
+import { getDateXDaysAgo } from "../utils/date-utils.js";
+import { parseArgs } from "node:util";
+
+const {
+    values: { provider }
+} = parseArgs({
+    options: { provider: { type: "string" } }
+});
+
+const start = '2024-04-07';
```

Comment:
_A minor detail, feel free to ignore._

Since we started collecting TTFB data only recently, should we change the start date to something like `2025-01-01`?

```suggestion
const start = '2025-01-01';
```